website/src/components/hero/index.js

```diff
@@ -12,6 +12,22 @@ function Hero({ heading, subheading, showGraphic = false, customStyles = {}, cla
           <div className={`col col--7 ${colClassNames ? colClassNames : ''}`}>
             <h1>{heading}</h1>
             <p>{subheading}</p>
+            {callToActionsTitle && <span className={styles.callToActionsTitle}>{callToActionsTitle}</span>}
+            {callToActions.length > 0 && (
+              <div className={styles.callToActions}>
+                {callToActions.map((c, index) => (
+                  c.href ? (
+                    <a key={index} href={c.href} className={`${styles.callToAction}`}>
+                      {c.label}
+                    </a>
+                  ) : (
+                    <a key={index} onClick={c.onClick} className={`${styles.callToAction}`}>
+                      {c.title}
+                    </a>
```

Comment:
I see `c.href` and `c.label` here which looks intended for being able to link out to urls from the ctas. However in the `quickstartGuideList` index file, it looks like only `title` and `value` are ever passed into the Hero component, into the `title` and `onClick` properties.

```
featuredGuides.map(guide => ({
    title: guide.title,
    onClick: () => handleCallToActionClick(guide.value)
})
```
I tried adding a new object to the `featuredGuides` array with `href` & `label` properties. With this combo the cta appears blank and still tries to run `onClick`. If i switch `label` to `title`, the cta text will show, but `onClick` is still run. 
![image](https://github.com/user-attachments/assets/38c09e33-ef5b-4ec6-a3ed-be1ccb207c27)

Should it be possible to pass the `href` property into the Hero component to allow linking to urls? Or should this only be limited to activating the guide filters?

Reply:
Ope good catch. Just pushed an update allowing the hero to accept the href as well. Added an example in there too, lmk what you think.